### PR TITLE
Evaluating flows in BaseFlowMeasurement and fixed evaluating infinity in NonzeroFlowEvaluator

### PR DESCRIPTION
**Pull request by enhaut**
Description:  
Replaced hardcoded result pass with evaluating in `BaseFlowMeasurement._report_flow_results()` to correctly evaluating flows with invalid duration.  
Fixed evaluating `inf` as valid passed result in `NonzeroFlowEvaluator.evaluate_results()`.

Reviews:  
@Axonis @olichtne @jtluka

### CODE REVIEW

File: lnst/RecipeCommon/Perf/Measurements/BaseFlowMeasurement.py
Comment (olichtne) [inline lnst/RecipeCommon/Perf/Measurements/BaseFlowMeasurement.py:306]: nitpick here... these are **metrics** or **measurement results**, not flows. all of these metrics are related to the same flow measurement.

**Review by olichtne — COMMENTED** [review]
I don't think my nitpick is imporant enough to block this PR but I'll wait for review from others before approval.

File: lnst/RecipeCommon/Perf/Measurements/BaseFlowMeasurement.py
Comment (enhaut) [reply lnst/RecipeCommon/Perf/Measurements/BaseFlowMeasurement.py]: Thanks, i just changed that.

**Review by olichtne — DISMISSED** [review]

**Review by olichtne — APPROVED** [review]

**Pull request merged**
